src/main/UseCases/UseCaseUpdate.java

```diff
@@ -0,0 +1,72 @@
+import java.util.*;
+
+public class UseCaseUpdate {
+
+    public void updateQuantity(String serial_num, Employee e, int modified_quantity) throws Exception {
```

Comment:
Instead of having separate local variables for every method, values like serial_num, Employee e, modified_quantity can be passed onto the constrictor for UseCaseUpdate